gramps/plugins/textreport/descendreport.py

```diff
@@ -12,7 +12,7 @@
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 2 of the License, or
-# (at your option) any later version.
```

Comment:
Was removal of this line intentional? If not, it should be reintroduced.

Reply:
It was not intentional, I've gone ahead and reintroduced the line. Thanks for pointing this out!